src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/FallbackWorker.java

```diff
@@ -0,0 +1,35 @@
+package edu.kit.ipd.crowdcontrol.objectservice.crowdworking;
```

Comment:
Does that really belong in the crowdworking package ? I mean its the rest api which defines that its the field email